pkg/meta/hooks.go

```diff
@@ -122,8 +123,8 @@ func OnDeleteManifest(repo, reference, mediaType string, digest godigest.Digest,
 			manageRepoMetaSuccessfully = false
 		}
 
-		if refferredDigest, hasSubject := common.GetReferredSubject(manifestBlob); hasSubject {
-			err := metaDB.DeleteReferrer(repo, refferredDigest, digest)
+		if referredDigest, hasSubject := common.GetReferredSubject(manifestBlob); hasSubject {
+			err := metaDB.DeleteReferrer(repo, referredDigest, digest)
```

Comment:
Please use a different variable, so the original err is reported. And update the value to wrap both errors in case a new one is returned here.